nio_cli/commands/buildspec.py

```diff
@@ -52,14 +53,16 @@ def _merge_previous_into_new_spec(self, previous_spec, spec):
                             get(field, {}).get(name, {}).items():
                         if attr not in spec[block][field][name]:
                             spec[block][field][name][attr] = value
+            keyorder = ['version', 'description', 'properties', 'inputs', 'outputs', 'commands']
+            spec[block] = OrderedDict(sorted(spec[block].items(), key=lambda i:keyorder.index(i[0])))
```

Comment:
properties, inputs, outputs and commands are all dictionaries too. Those should be sorted as well. Maybe alphabetically?